docs/deployment.md

```diff
@@ -4,25 +4,10 @@ General Django deployment considerations apply to deploying Cove. We deploy usin
 
 ## Before a live deploy
 
-Travis tests will fail if a branch isn't ready to be merged and deployed. This includes if OCDS translations are missing.
+Travis tests will fail if a branch isn't ready to be merged and deployed. This may happen if translations are missing.
```

Comment:
"This may happen if translations are missing." I don't think we do any translation tests any more (360 and IATI don't have any) so I think this line should be removed

Reply:
That's correct.